resources/windows/wix/registryEntries.wxs.erb

```diff
@@ -1,15 +1,44 @@
 <?xml version='1.0' encoding='windows-1252'?>
 <Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+  <!--
+    Registry Entries
+    This is a suggested set of entries that can be used to record the Installation Directory for upgrade installs.
+    Any definitions in this will be over-ridden if the same file name is used in the Product Specific resouces directory.
+  -->
   <Fragment>
     <ComponentGroup Id="RegistryComponentGroup">
-        <Component Id="RegistryEntriesArchitectureDependent" Directory="TARGETDIR" Guid="E6D5AF4F-ACC4-4D11-AFCE-299A9ED2152C" Win64="<%= settings[:win64] %>" Permanent="yes">
-          <RegistryKey Root="HKLM" Key="SOFTWARE\<%= settings[:company_name] %>\<%= settings[:product_name] %>" ForceCreateOnInstall="yes" >
-            <RegistryValue Type="integer" Value="0"/>
+        <Component
+          Id="RegistryEntriesArchitectureDependent"
+          Directory="TARGETDIR"
+          Guid="E6D5AF4F-ACC4-4D11-AFCE-299A9ED2152C"
```

Comment:
Actually, now that I think about it, for a generalized file in vanagon, the guid description at http://wixtoolset.org/documentation/manual/v3/xsd/wix/component.html makes me think maybe this should be "*"